src/common/ApprovalsChip.spec.tsx

```diff
@@ -35,36 +36,12 @@ describe('ApprovalsChip', () => {
   it('renders an ApprovalsChip component with 2/2 approvals', async () => {
     render(
       <ThemeProvider theme={lightTheme}>
-        <ApprovalsChip approvals={['Accepted', 'Accepted']} />
+        <ApprovalsChip approvals={[{ reviewer: 'Alice', status: ApprovalStates.Accepted},  { reviewer: 'Bob', status: ApprovalStates.Accepted }]} />
       </ThemeProvider>
     )
 
     await waitFor(async () => {
       expect(await screen.findByText('Approvals 2/2')).not.toBeUndefined()
     })
   })
-
```

Comment:
Can we re-add the deleted tests with the new props? Because the component has different states it would be nice to make sure other cases are displaying correctly.